policyengine_us/variables/gov/states/il/tax/income/exemptions/il_total_exemptions.py

```diff
@@ -0,0 +1,19 @@
+from policyengine_us.model_api import *
+
+
+class il_total_exemptions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "IL total exemption allowance"
+    unit = USD
+    definition_period = YEAR
+    reference = ""
+    defined_for = "il_is_exemption_eligible"
+
+    formula = sum_of_variables(
```

Comment:
And elsewhere - this will get it working with the new policyengine-core after a `git pull upstream master`
```suggestion
    adds = [
            "il_personal_exemption",
            "il_aged_blind_exemption",
            "il_dependent_exemption",
        ]
```